e2e-tests/tfm/cypress/e2e/journeys/facility/facilities.spec.js

```diff
@@ -56,31 +56,38 @@ context('Facility page', () => {
 
   it('renders all facilities from TFM', () => {
     cy.visit(relative('/facilities'));
-    cy.url().should('eq', relative('/facilities'));
-
-    page.facilitiesPage.ukefFacilityIdColumn().invoke('attr', 'aria-sort').should('contain', 'ascending');
-    page.facilitiesPage.productColumn().should('contain', 'Product');
-    page.facilitiesPage.dataTypeColumn().should('contain', 'Type');
-    page.facilitiesPage.exporterColumn().should('contain', 'Exporter');
-    page.facilitiesPage.valueColumn().should('contain', 'Value (export currency)');
-    page.facilitiesPage.coverEndDateColumn().should('contain', 'Cover end date');
-    page.facilitiesPage.facilityStageColumn().should('contain', 'Facility stage');
-
-    cy.get('tr:nth-child(n+1) [data-cy="facility__ukefFacilityId"]').contains('1000000');
-    cy.get('tr:nth-child(n+1) [data-cy="facility__product"]').contains(dealTwo.dealType);
-    cy.get('tr:nth-child(n+1) [data-cy="facility__type"]').contains(dealTwoFacilities[1].type);
-    cy.get('tr:nth-child(n+1) [data-cy="facility__companyName"]').contains(dealTwo.exporter.companyName);
-    cy.get('tr:nth-child(n+1) [data-cy="facility__facilityValue"]').contains('GBP 1,234.00');
-    cy.get('tr:nth-child(n+1) [data-cy="facility__coverEndDate"]').contains('24 Sep 2020');
-    cy.get('tr:nth-child(n+1) [data-cy="facility__facilityStage"]').contains('Unissued');
-
-    cy.get('tr:nth-child(n+2) [data-cy="facility__ukefFacilityId"]').contains('1000000');
-    cy.get('tr:nth-child(n+2) [data-cy="facility__product"]').contains(dealTwo.dealType);
-    cy.get('tr:nth-child(n+2) [data-cy="facility__type"]').contains(dealTwoFacilities[0].type);
-    cy.get('tr:nth-child(n+2) [data-cy="facility__companyName"]').contains(dealTwo.exporter.companyName);
-    cy.get('tr:nth-child(n+2) [data-cy="facility__facilityValue"]').contains('GBP 1,234,567,890.1');
-    cy.get('tr:nth-child(n+2) [data-cy="facility__coverEndDate"]').contains(dateConstants.oneMonthFormattedShort);
-    cy.get('tr:nth-child(n+2) [data-cy="facility__facilityStage"]').contains('Issued');
+    cy.url().should('eq', relative('/facilities/0'));
+
+    pages.facilitiesPage.facilitiesTable.headings.ukefFacilityId().contains('Facility ID');
+    pages.facilitiesPage.facilitiesTable.headings.ukefFacilityId().invoke('attr', 'aria-sort').should('eq', 'ascending');
+    pages.facilitiesPage.facilitiesTable.headings.dealType().contains('Product');
+    pages.facilitiesPage.facilitiesTable.headings.type().contains('Type');
+    pages.facilitiesPage.facilitiesTable.headings.companyName().contains('Exporter');
+    pages.facilitiesPage.facilitiesTable.headings.value().contains('Value (export currency)');
+    pages.facilitiesPage.facilitiesTable.headings.coverEndDate().contains('Cover end date');
+    pages.facilitiesPage.facilitiesTable.headings.facilityStage().contains('Facility stage');
+
+    for (let i = 0; i < 4; i += 1) {
```

Comment:
whats going on here?

Reply:
`.contains` matches substrings, so these assertions test that all of the facility IDs start with '1000000'. Ordering is testing elsewhere (`facilities-pagination.spec.js`)